generators/client/templates/react/src/test/javascript/spec/app/shared/auth/private-route.spec.tsx.ejs

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import { Route } from 'react-router-dom';
-import { shallow } from 'enzyme';
+import { Router } from 'react-router-dom';
+import { shallow } from 'enzyme'; import { createMemoryHistory } from 'history';
```

Comment:
you forgot to remove enzyme here (+ the new line)

Reply:
yes I correct